src/Components/ComponentPins/BottomPageCheckSvg.js

```diff
@@ -0,0 +1,169 @@
+    agSBs3MGLvxugezNU4Q81UoXv5RyJeuLxGfRzJ2Dv3R75tQL1laqV9HinjihbWEJeYqQp1oFebzZ
+    ii7eFjt919/ZFMnPCHgPetGRichhi8afv2D72/gDHgh6ipCnWuXiV+Di/XYFOIvmMwM+5OhFL0wJ
+    uHjb6Mz2wLFtikNHtVEUIU+1ysXjqtYXADw6+GvMoDkX6P29Btz7xnaztC2KsXXDtg1FyFOtcfFr
+    KbcNxl0l/Tz4JouLPquj7wHobSdLOxf5ScrtiQ30fUKeurS4MIO6pov3Wwe/SblVAW709dkdfBXo
+    bRWvjVjss9r9hrkc7pZ5a5+VIuQp6v+qAt+rlJuN2VYFt+Tgq0YtK/d53+CChgu7bvGzUoQ8RVW2
+    aibOwYc2G7tV8OHWBzZymbrPO7vxUQtFyFNU8KSnAvL+fFbcrhdXs976hc23qGyLYrz34EcuBD1F
+    yFM3Bz2cdTJ3sLMTnW5pNs2pnxuPMZzK4UlWfqM1iiLkqZt0tAj5GYAutGVB2z67uXk8eMSfZkUn
+    TxHy1M3CDs9tRdDh2ahbuf6mY9dy83jTeQ6Bn39HwFOEPHWrbnbr2hb+RKXPsPHYNdz8CkYzIcjT
+    zVOEPHVzoPM3XpfQpggBrq0jGTykHC9yfiRDwFOEPHVT2jvIVR2bJy0FfexC5w8Av8QJVxRFyFMX
+    B1zojFQDW9vbFB7y2LZZSXvWBlCEPNVBwPu+9A6c+07avwgoi4B+A6OadWA0Q1HvybOzQJ4ug/oV
+    4KOrrzobta2ffedA79tVdPLUqextnC+9a/4xqrOOvupc1K58D7vIqMafHUtRdaPD7BQO906Eeub+
+    GEU1zZmsg/njRzL7gHtnHVHvBX3IHGRNIJ+xQKkLJ2PW0e9kXzHCoagm9bR3xqE2j3oNh9p79+ZM
+    Tuo9rl7qXEeHHVjXvhPqtNzBkaCPypFhr8EQM9RLJOipJnD3rYou5w1HN9R7FZqOHFpvEsylXs2w
+    AOf42pviGxP0VFVrImQUujabpAruBD1VlzuZHC8sXEHE1ltkIchXAR73HrE3JuipUEJmEl8EFcqZ
+    rtyAlZqLHufKU1WAt91c/WZ3S6nZAymteGM8xgy3iC2iOJB4JOWJ83jB4KHE3Qa8wb1uO4OuOFjf
+    /tzCxQ8f7flEa4l1xFryx2fi6Wr+TIbocZJp5I/sAPJ4ok9xLmdx+vxYytPmCw31Z0zOUJIycW/f
+    jVa1IkIbcs3dKLBLOy+ig8eRjd+wzS6A/UD9sI66WUse8DNgsJ2PbCeN+bMJGkF+D8lob/6iDv5e
+    oZ5qstnrh5CkSU2SUu1IUu868Kg/PAUKXYdv9bX5O0rkeG99u/DN9HuZak0N4PVomFhH3awlMwQL
+    zZNXZbDFawDyuyaQD7VrZnrVGGsgzNGhjMDdh4acTNJ2ORCcTrtzECuS7wfEm5SHdlcOLVs69A6d
+    dVsU6VMeD1pTVk8jV2Oso27WEh4GX9TPcx7fNJ61riaaT2upOGWsyslju2aiiTcAl5G5oflYyj69
+    vYZ9+na3IPaBpJxrUn6PJGTl0LKF2keK9kHKtmeir1mpq2cdsZbQEBT183ceXzWspqYwOo6OjNMG
+    7mOpb9YHd5HJ4Y1ZcyR30Mpp0qOnbndoGeszT9WlftfE/KY/vyngLCHbuo98zDRt3PeDLc+dPvcb
+    64i1BLVkhuAHOPmvUE+1Lj4G+ZCbx/6gPbeGK40l571r29CFtN95bGA0Z714TMwiXjRPvOvIOvJ9
+    4USGmZSz0tDBT1lHrCU5nLRgtWT9eKulmItv7OT9/8BW/yg+t4UrTTEML2bcPELLZgiJjHPxmaC3
+    n5SYmN554A1XS85XzZGphOf0duF7wxuvWBNollhHrCVfSzOopTdXS41bn2mDYWbV0NMgb+5jDA4k
+    lJxM0NtPylhiWuvOknMKMXeuo4vbDePIWMC9rVhHrKUGtTSDWloGAP8uyIdAH7spWyTlHbiP0DCT
+    ao/8EBOTcwnDTTzP1ADf9qmTp9TRjnXEWjqhluz3fgFdZT01dQMJJFqqMdBExBjA8/b6hK6jle0H
+    v4LTZlr5vY7sua4sgGIdUZ+qlpITExST1GbbpPCYwnNczNHeoabfV2Mvh6cd+ZOPuGsp64j6oFpK
+    fiFJMVF9JHK88pVqnwMJnXwUO8yAG9ixjqgPqKXkFxLUPyYVSckEbZ8DEWm+bS4BzzqiPqiWkjMm
+    aixxqW4MOYVQZx1Rn6+WkislLNUNN0KxjijWEkVRFEVRFEVRv6z/CTAAXSQZYODEjjQAAAAASUVO
+    RK5CYII="
+        transform="matrix(1 0 0 1 2.4961 2.4961)"
+      />
+      <g>
+        <g>
+          <path
+            className="st9"
+            d="M9.1,8.8v40.6c0,6.2,5.1,11.3,11.3,11.3h124c9.7-1.3,8.5-8.3,8.5-12.2c0-18.8,15.3-34.1,34.1-34.1
+                    c18.8,0,34.1,15.3,34.1,34.1c0,4-1.3,10.9,8.4,12.2h125.1c6.2,0,11.3-5.1,11.3-11.3V8.8H9.1z"
+          />
+        </g>
+      </g>
+    </g>
+    <g
+      onClick={() => dispatch({
+        type: 'NEXT_PAGE',
+        page: page + 1,
```

Comment:
```page + 1``` must be in the reducer not in the action

Reply:
Done